tests/SprykerEcoTest/Zed/Unzer/Business/ExecutePostSaveHookFacadeTest.php

```diff
@@ -7,55 +7,296 @@
+//    ];
+//
+//    /**
+//     * @var array
+//     */
+//    protected const UNZER_MARKETPLACE_PAYMENT_METHODS = [
+//        UnzerConfig::PAYMENT_METHOD_KEY_MARKETPLACE_BANK_TRANSFER,
+//        //*UnzerConfig::PAYMENT_METHOD_KEY_MARKETPLACE_CREDIT_CARD,
+//        UnzerConfig::PAYMENT_METHOD_KEY_MARKETPLACE_SOFORT,
+//    ];
+
     /**
-     * @var array
+     * @return void
      */
-    protected const UNZER_STANDARD_PAYMENT_METHODS = [
-        // will be implemented along with regular payments
-    ];
+    public function testExecutePostSaveHookReturnsExternalRedirectWhileUnzerBankTransferIsGiven(): void
+    {
+        //Arrange
+        $unzerPaymentTransfer = $this->tester
+            ->createUnzerPaymentTransfer(false, false);
+        $paymentTransfer = $this->tester
+            ->createPaymentTransfer(UnzerConfig::PAYMENT_METHOD_KEY_BANK_TRANSFER)
+            ->setUnzerPayment($unzerPaymentTransfer);
+        $quoteTransfer = $this->tester
+            ->createQuoteTransfer()
+            ->setPayment($paymentTransfer);
+        $checkoutResponseTransfer = $this->tester->createCheckoutResponseTransfer();
+
+        $saveOrderTransfer = $this->tester->haveOrder(
+            [
+                'unitPrice' => 72350,
+                'sumPrice' => 72350,
+                'orderReference' => 'DE--1',
+            ],
+            'UnzerBankTransfer01',
+        );
+
+        $this->tester->haveUnzerEntities($quoteTransfer, $saveOrderTransfer);
+
+        //Act
+        $this->tester->getFacade()->executePostSaveHook($quoteTransfer, $checkoutResponseTransfer);
+
+        //Assert
+        $this->assertTrue($checkoutResponseTransfer->getIsExternalRedirect());
+        $this->assertSame(UnzerZedTester::UNZER_REDIRECT_URL, $checkoutResponseTransfer->getRedirectUrl());
+    }
 
     /**
-     * @var array
+     * @return void
      */
-    protected const UNZER_MARKETPLACE_PAYMENT_METHODS = [
-        UnzerConfig::PAYMENT_METHOD_KEY_MARKETPLACE_BANK_TRANSFER,
-        UnzerConfig::PAYMENT_METHOD_KEY_MARKETPLACE_CREDIT_CARD,
-        UnzerConfig::PAYMENT_METHOD_KEY_MARKETPLACE_SOFORT,
-    ];
+    public function testExecutePostSaveHookReturnsExternalRedirectWhileUnzerSofortIsGiven(): void
+    {
+        //Arrange
+        $unzerPaymentTransfer = $this->tester
+            ->createUnzerPaymentTransfer(false, false);
+        $paymentTransfer = $this->tester
+            ->createPaymentTransfer(UnzerConfig::PAYMENT_METHOD_KEY_SOFORT)
+            ->setUnzerPayment($unzerPaymentTransfer);
+        $quoteTransfer = $this->tester
+            ->createQuoteTransfer()
+            ->setPayment($paymentTransfer);
+        $checkoutResponseTransfer = $this->tester->createCheckoutResponseTransfer();
+
+        $saveOrderTransfer = $this->tester->haveOrder(
+            [
+            'unitPrice' => 72350,
+            'sumPrice' => 72350,
+            'orderReference' => 'DE--2',
+            ],
+            'UnzerSofort01',
+        );
```

Comment:
```suggestion
        $saveOrderTransfer = $this->tester->haveOrder(
            [
                'unitPrice' => 72350,
                'sumPrice' => 72350,
                'orderReference' => 'DE--2',
            ],
            'UnzerSofort01',
        );
```

Please, check similar cases below

Reply:
Fixed.